notebooks/Read_NectarCAM_events.ipynb

```diff
@@ -19,7 +19,7 @@
     "from matplotlib import pyplot as plt\n",
```

Comment:
<p>I don't see any difference with the original notebook here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/nectarchain/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>